en/docs/get-started/streaming-quick-start-guide.md

```diff
@@ -1,142 +1,221 @@
 # Quick Start Guide - Streaming Integration
 
-This quick start guide gets you started with the Streaming Integrator (SI), in just 5 minutes.
+Let's get started with WSO2 Streaming Integrator(SI) by running a simple streaming use case in your local environment. 
 
-In this guide, you will download the SI distribution as well as Kafka, and then try out a simple Siddhi application. This Siddhi application consumes messages from a file processes the data, generates an output, and then publishes that output to a Kafka topic.
+## Before you begin...
 
-The following is the outline of this quick start guide.
+1. Install [Oracle Java SE Development Kit (JDK)](http://java.sun.com/javase/downloads/index.jsp) version 11 or 1.8 and set the `JAVA_HOME` environment variable.
+   For more information on setting the `JAVA_HOME` environment variable for different operating systems, see [Setup and Install]({{base_path}}/install-and-setup/install/installing-the-product/installing-api-m-runtime/).
+2. Download the Streaming Integrator and Streaming Integrator Tooling distributions from the [WSO2 Streaming Integrator site](https://wso2.com/integration/streaming-integrator/) and extract them to a location of your choice. Hereafter, the extracted location is referred to as `<SI_HOME>` and `<SIT_HOME>` respectively.<br/><br/>
```

Comment:
Why are we downloading SI and not API-M?

Reply:
yes